src/portal/src/app/app.module.ts

```diff
@@ -20,12 +20,21 @@ import { HarborRoutingModule } from './harbor-routing.module';
 import { SharedModule } from './shared/shared.module';
 import { AccountModule } from './account/account.module';
 import { ConfigurationModule } from './config/config.module';
+import { registerLocaleData } from '@angular/common';
 
 import { TranslateService } from "@ngx-translate/core";
 import { AppConfigService } from './app-config.service';
-import {SkinableConfig} from "./skinable-config.service";
+import { SkinableConfig } from "./skinable-config.service";
 import { ProjectConfigComponent } from './project/project-config/project-config.component';
 
+import zh from '@angular/common/locales/zh-Hans';
```

Comment:
I also saw this [github issue ](https://github.com/angular/angular/issues/20286), it imported class is a lit bit different.

Reply:
I mean the name of the imported class, i.e. "registerLocaleData(*localeFr*, 'fr-FR', localeFrExtra);"